personal_profile/views.py

```diff
@@ -1,5 +1,11 @@
 from django.shortcuts import render
+from django.views.generic import DetailView
+from .models import PersonalProfile
 
 
 def personal_profile(request):
-    return render(request, 'personal_profile.html')
```

Comment:
You can remove this `personal_profile` function entirely

Reply:
thank you for all the comments, done